src/openrct2/ride/VehicleSubpositionData.cpp

```diff
@@ -18527,396 +18536,396 @@ CREATE_VEHICLE_INFO(TrackVehicleInfo_9395FF, {
 })
 
 CREATE_VEHICLE_INFO(TrackVehicleInfo_939691, {
-    { -32768,    0,    0,  0,  0,  0 }, {     31,   16,    0,  0,  0,  0 }, {     29,   16,    0,  0,  0,  0 }, {     27,   16,    0,  0,  0,  0 }, { -32768,    4,    1,  0,  0,  0 },
```

Comment:
What do the occurrences that were not replaced with either MINI_GOLF_STATE or MINI_GOLF_ANIMATION signify?

Reply:
They are normal sub position entries that move the vehicle the set distance